<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OmniSlim-tree 2D</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0">
    <item>
     <widget class="QGraphicsView" name="slimtreeView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="queryList"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="TreeCapacity">
          <property name="text">
           <string>Tree Capacity (C):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="CapacitySpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>25</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="NumPuntos">
          <property name="text">
           <string>Nº de Puntos: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="NumPuntosSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="FociLabel">
          <property name="text">
           <string>Nº de Foci:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="FociSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="BuildLabel">
          <property name="text">
           <string>Build Dist. Calc.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="BuildEdit">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="PostLabel">
          <property name="text">
           <string>Post Dist. Calc.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="PostEdit">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="GenerarPuntosButton">
        <property name="text">
         <string>Generar Puntos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SlimDownButton">
        <property name="text">
         <string>Slim-down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FociButton">
        <property name="text">
         <string>Calcular Foci</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="LimpiarButton">
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="GenDataButton">
        <property name="text">
         <string>Generar Datos</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
